lib/helpers/sassNodeModulesAliasImporter.js

```diff
@@ -0,0 +1,49 @@
+var path = require("path");
+var findNodeModules = require("find-node-modules");
+var fs = require("fs");
+
+var NODE_MODULES_ALIAS = "~";
+var sassFileExtensions = [".sass", ".scss"];
+
+module.exports = function sassNodeModulesAliasImporter(config) {
+	if (typeof config.sassNodeModulesAlias === "string") {
+		NODE_MODULES_ALIAS = config.sassNodeModulesAlias;
+	}
+
+	return function sassNodeModulesAliasImporter(url, prev) {
+		if (url.substr(0, NODE_MODULES_ALIAS.length) === NODE_MODULES_ALIAS) {
+			var i, j, k;
```

Comment:
Make this more readable, please. Make the variable names understandable and define them when it makes sense.